runtime/src/accounts_db.rs

```diff
@@ -7646,11 +7646,16 @@ impl AccountsDb {
         skipped_rewrites: &Rewrites,
     ) {
         let mut skipped_rewrites = skipped_rewrites.read().unwrap().clone();
+        let filler_accounts_enabled = self.filler_accounts_enabled();
+        if skipped_rewrites.is_empty() && !filler_accounts_enabled {
```

Comment:
Did you add the `&& !filler_accounts_enabled` because of the `simulate the time ...` comment? If rewrites is empty, would we not want our simulated time to match our actual (none) time in this function?

Reply:
doh. you're right... ha